src/Tizen.Wearable.CircularUI.Forms.Renderer/ContentButtonRenderer.cs

```diff
@@ -0,0 +1,155 @@
+        EButton _button;
+
+        SmartEvent _pressed;
+        SmartEvent _unpressed;
+        SmartEvent _clicked;
+
+        protected override void OnElementChanged(ElementChangedEventArgs<ContentButton> e)
+        {
+            if (Control == null)
+            {
+                var box = new NBox(XForms.NativeParent)
+                {
+                    MinimumWidth = _defaultMinimumSize,
+                    MinimumHeight = _defaultMinimumSize
+                };
+                box.LayoutUpdated += OnLayout;
+
+                _button = new EButton(XForms.NativeParent);
+                _button.BackgroundColor = EColor.Transparent;
+                _button.SetPartColor("effect", EColor.Transparent);
+                _button.SetPartColor("effect_pressed", EColor.Transparent);
+                _button.Show();
+
+                _pressed = new SmartEvent(_button, "pressed");
+                _unpressed = new SmartEvent(_button, "unpressed");
+                _clicked = new SmartEvent(_button, "clicked");
+
+                _pressed.On += OnPressed;
+                _unpressed.On += OnReleased;
+                _clicked.On += OnClicked;
+
+                box.PackEnd(_button);
+
+                SetNativeControl(box);
+            }
+
+            UpdateContent();
+            base.OnElementChanged(e);
+        }
+
+        protected override void Dispose(bool disposing)
+        {
+            if (disposing)
+            {
+                _pressed.On -= OnPressed;
+                _unpressed.On -= OnReleased;
+                _clicked.On -= OnClicked;
+            }
+            base.Dispose(disposing);
+        }
+
+        protected override void OnElementPropertyChanged(object sender, PropertyChangedEventArgs e)
+        {
+            if (e.PropertyName == ContentButton.ContentProperty.PropertyName)
+            {
+                UpdateContent();
+            }
+            base.OnElementPropertyChanged(sender, e);
+        }
+
+        protected override Xamarin.Forms.Size MinimumSize()
+        {
+            var size = base.MinimumSize();
+            var height = XForms.ConvertToScaledPixel(Element.MinimumHeightRequest);
+            var width = XForms.ConvertToScaledPixel(Element.MinimumWidthRequest);
+
+            size.Width = (size.Width > width) ? size.Width : width;
+            size.Height = (size.Height > height) ? size.Height : height;
+
+            return size;
+        }
+
+        void OnPressed(object sender, EventArgs args)
+        {
+            Element?.SendPressed();
+        }
+
+        void OnReleased(object sender, EventArgs args)
+        {
+            Element?.SendReleased();
+        }
+
+        void OnClicked(object sender, EventArgs args)
+        {
+            Element?.SendClicked();
+        }
+
+        void OnLayout(object sender, LayoutEventArgs args)
+        {
+            if (Element.Content != null)
+            {
+                Element.Content.Layout(args.Geometry.ToDP());
```

Comment:
It will pass Absolute coordinate into Core layer
```suggestion
                _content.Geometry = NativeView.Geometry;
```
I think, it is more correct